### PR TITLE
PTM list without multiselect

### PR DESCRIPTION
**Pull request by apmody**

### CODE REVIEW

**Comment by jamesaoverton** [comment]
Looking at the dev server https://tetramers.ontodev.com

- The CSS for the Typeahead needs a bit of a border or padding, and is often not wide enough for the modification names
- Please move the Submit button below the form fields
- The error messages should be more prominent, using the [Bootstrap `.alert-danger` class](https://getbootstrap.com/docs/4.5/components/alerts/)
- Likewise the success message should use `.alert-success`
- There's no need for the "Please enter info" message, please remove it
- Please remove the "Installation for command line interface" section, since it does not contain useful information yet
- It would be good to add a Reset button to clear the fields (sorry, I didn't think of this before); that can wait, if you prefer

File: tetramer_validator/data/PTM_list.csv [hunk 1]
```diff
@@ -0,0 +1,25 @@
+display_name,synonyms
```
Comment (jamesaoverton) [inline tetramer_validator/data/PTM_list.csv:1]: In general I prefer TSV. It's likely that some of these PTMs will contain commas, which requires quoting or escaping.

**Review by jamesaoverton — CHANGES_REQUESTED** [review]
In addition to the interface changes requested above, I took a quick look at the code. The only change I'd like right now is to use TSV format instead of CSV throughout.

File: tetramer_validator/data/PTM_list.csv (hunk 1 above)
Comment (apmody) [reply tetramer_validator/data/PTM_list.csv:1]: Would it be better to store it in some sort of JSON format?

Comment (jamesaoverton) [reply tetramer_validator/data/PTM_list.csv:1]: If that makes the other code simpler, then fine. In general tables are easier to work with than JSON, so tables should be preferred for “flat” data.

**Review by jamesaoverton — APPROVED** [review]
Please use this CSS for Typeahead https://github.com/bassjobsen/typeahead.js-bootstrap4-css/blob/master/typeaheadjs.css

Then you can merge this branch.

**Pull request merged**